android/build.gradle

```diff
@@ -0,0 +1,10 @@
+buildscript {
```

Comment:
Do we also need this file? We have one in `android/app/` as well...

Reply:
Without linking the ADP here, the project won't be configured correctly. When `android/app` was the only module in the project, it was sufficient to link the plugin just at the module level. Once I've added the `support` module, I had to lift this declaration to the project level, otherwise, two instances of the plugin get into a conflict. 

I should explore if removing the `buildscript` block from the `android/app` can break existing integrations.
 - [ ] Check if the `buildscript` block can be removed safely from the `android/app`.

Reply:
I checked and removed the dupe entry for applying ADP, but the buildscript block still has to be there for the kotlin gradle plugin.